spec/acceptance/courses/enrollments_request_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+require "rspec_api_documentation/dsl"
+
+RSpec.resource "Api::V1::Courses::Enrollments" do
+  include_context "with authorized API request"
+
+  let(:course_id) { course.id }
+  let(:course) { create(:course) }
+
+  get "/api/v1/courses/:course_id/enrollment" do
+    context "when user enrolled for the given course" do
+      before { create(:enrollment, course: course, user: current_user) }
+
+      example_request "shows enrollment object" do
+        expect(status).to eq(200)
+        expect(response).to include(
```

Comment:
Did you decide to check the response this way?

Reply:
Yes, I like this option, also `include` keyword represents the idea that the data in reponse is guaranteed.

Reply:
I used that approach you suggested me before but maybe without creating another variable `expected_response`.
I don't like fixtures because when I use them I have to open several files to read 1 spec (like we do in Rewards).